spi/common/web-spi/src/main/java/org/eclipse/edc/web/spi/configuration/DefaultWebServiceConfigurer.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright (c) 2020-2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.web.spi.configuration;
+
+
+import org.eclipse.edc.spi.system.ServiceExtensionContext;
+import org.eclipse.edc.web.spi.WebServer;
+
+import static java.lang.String.format;
+
+public class DefaultWebServiceConfigurer implements WebServiceConfigurer {
```

Comment:
`WebServiceConfigurerImpl` as `Default` should be used when there are multiple implementations